test/test_eos.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import division, absolute_import, print_function
+
+__copyright__ = (
+    """Copyright (C) 2020 University of Illinois Board of Trustees"""
+)
```

Comment:
The parens are weird to me.  If you'd like multiple lines, just do a triple-quoted string.